src/Components/BreathingRoom/BreathingRoom.js

```diff
@@ -1,11 +1,41 @@
 import React from 'react';
+import {Component} from 'react'
 import './BreathingRoom.scss';
 
-const BreathingRoom = () => {
+import {inhaleExhale} from './BreathingRoomAnimation.js'
 
-  return (
-    <h1>Breathing Room</h1>
+class BreathingRoom extends Component {
+  constructor(props){
+    super(props)
+    this.state ={
+      isBreathing: false
+    }
+  }
+  clickHandler = () => {
+    this.setState({isBreathing: true})
+    inhaleExhale()
+  }
+
+  render(){
+    let instructions = (
+      <section className='directions'>
+      <h1>Time to Take Breathe</h1>
+      <p>Follow the animation, breathe in as it expands
```

Comment:
I wonder if there's a way to incorporate these instructions into the animation itself. If we could have "breathe in"/"breathe out" display as the animation moves it might be a bit more intuitive. Doesn't need to be a priority, but might be something to pay with if we have time.